laboratory/.github/workflows/playwright.yml

```diff
@@ -0,0 +1,27 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [ main, master ]
+  pull_request:
+    branches: [ main, master ]
+jobs:
+  test:
+    timeout-minutes: 60
```

Comment:
I feel like we could get away with 30 or even 15 here, could we set this a little lower so we don't waste minutes in the case of a coding error?